src/us/kbase/workspace/database/GetObjectInformationParameters.java

```diff
@@ -171,4 +171,19 @@ public int getLimit() {
 	public boolean asAdmin() {
 		return asAdmin;
 	}
+	
+	/** Check if there are no filters set other than the object ID filters. The object ID filters
+	 * may or may not be set.
+	 * The other filters are the two date filters, the metadata filter, the savers filter, and
+	 * the type filter.
+	 * @return true if no filters other than the object ID filters are set.
+	 */
+	public boolean isObjectIDFiltersOnly() {
```

Comment:
It's used elsewhere in the code to ensure that only object id filters are active before doing a sort on the ws/obj/ver numbers. Other filters being active *could* force the sort to occur in memory.